spec-tests/src/main/java/org/candlepin/spec/bootstrap/data/builder/Contents.java

```diff
@@ -78,7 +78,8 @@ public static ContentDTO copy(ContentDTO source) {
      *  a ContentDTO instance with the required fields populated with semi-random values
      */
     public static ContentDTO random() {
-        String cid = StringUtil.random(18, StringUtil.CHARSET_NUMERIC);
+        // replace is needed because this id is used in a OID. BC does not allow leading zeros in a segment
+        String cid = StringUtil.random(18, StringUtil.CHARSET_NUMERIC).replace('0', '1');
```

Comment:
Given how this works, rather than do a replacement at the end, change the charset to a custom one that doesn't have a zero in it:
```
String cid = StringUtil.random(18, "123456789");
```

Alternatively, the cid could be prefixed with a 1 to avoid the need to do either:
```
String cid = StringUtil.random("1", 17, StringUtil.CHARSET_NUMERIC);
```